pkg/controller/common/version/version.go

```diff
@@ -130,6 +132,32 @@ func (v *Version) IsAfter(other Version) bool {
 		(v.Major == other.Major && v.Minor == other.Minor && v.Patch > other.Patch)
 }
 
+// MinInPods returns the lowest version parsed from labels in the given Pods.
+func MinInPods(pods []corev1.Pod, labelName string) (*Version, error) {
```

Comment:
Should this also replace the code here: 
https://github.com/elastic/cloud-on-k8s/blob/764c4fb41cf8721d911d2e83aee481bb2a9095f0/pkg/controller/elasticsearch/label/label.go#L85-L101

?

Reply:
Oh right! I totally missed this.